### DISCUSSION TITLE
Alot of 404 errors on documentation

### DISCUSSION THREAD

**Discussion by mavenmgofficial** [woocommerce/woocommerce#45027] [WooCommerce, Blocks, and Site Editing]
Hello,

I have been trying to find a way to make a customer confirm their order after clicking the “place order” button. The order would only be placed on the server side after they click “confirm” on the pop up generated. It seems like it shouldn’t be an issue, but the documentation has a bunch of links to 404 errors, specifically the Extensibility on the Server Side ( Hooks ) section. This is the link I’m referencing: [https://developer.woocommerce.com/2021/11/09/available-extensibility-interfaces-for-the-cart-and-checkout-blocks/](https://developer.woocommerce.com/2021/11/09/available-extensibility-interfaces-for-the-cart-and-checkout-blocks/)

**Comment by gigitux** [comment]
Thanks for opening the thread. We updated and fixed all the links.
